charts/management-portal/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: "2.0.0"
+appVersion: "2.1.1"
```

Comment:
If you want to use appVersion to to change image of the application please also make the following changes to the chart so it would actually be applied:
https://github.com/RADAR-base/radar-helm-charts/blob/ac068be59b78e7300586b5ac4868283e52c28de1/charts/data-dashboard-backend/templates/deployment.yaml#L35
https://github.com/RADAR-base/radar-helm-charts/blob/ac068be59b78e7300586b5ac4868283e52c28de1/charts/data-dashboard-backend/values.yaml#L14

Reply:
I don't necessarily want/need it to, just figured we should update it to reflect the appversion?

If youn think we should change, let me know